.github/workflows/publish-embedded-packages.yaml

```diff
@@ -0,0 +1,228 @@
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id || github.run_id }}
+          name: build-output-embedded
+          path: embedded/web/dist
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .node-version
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Publish npm
+        working-directory: embedded/web
+        run: |
+          npm version ${{ env.VERSION }} --no-git-tag-version
+          echo "ARTIFACT_VERSION=$(jq '.version' --raw-output package.json)" >> "$GITHUB_ENV"
+          npm publish --provenance --access public ${{ env.DRY_RUN == 'true' && '--dry-run' || '' }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN }}
+
+      - name: Add release note
+        if: ${{ env.DRY_RUN == 'false' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          append_body: true
+          body: |
+            ## NPM embedded package
+
+            This package provides a build of Element Call that can be used as an embedded widget within another application.
+
+            ```
+            npm install @element-hq/element-call-embedded@${{ env.ARTIFACT_VERSION }}
+            ```
+
+  publish_android:
+    needs: build_element_call
+    if: always()
+    name: Publish Android AAR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: 📥 Download built element-call artifact
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id || github.run_id }}
+          name: build-output-embedded
+          path: embedded/android/lib/src/main/assets/element-call
+
+      - name: ☕️ Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: "17"
+
+      - name: Get artifact version
+        run: echo "ARTIFACT_VERSION=${VERSION:1}" >> "$GITHUB_ENV"
+
+      - name: Publish AAR
+        if: ${{ env.DRY_RUN == 'false' }}
+        # TODO: can we do some kind of dry run?
+        env:
+          EC_VERSION: ${{ env.ARTIFACT_VERSION }}
+          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_RELEASE_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_RELEASE_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.GPG_SIGNING_KEY }}
+          ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.GPG_SIGNING_KEY_PASSWORD }}
+        run: embedded/android/publish_android_package.sh -s
+
+      - name: Add release note
+        if: ${{ env.DRY_RUN == 'false' }}
+        uses: softprops/action-gh-release@v2
```

Comment:
Should be pinned to avoid supply chain attack

Reply:
```suggestion
        uses: softprops/action-gh-release@c95fe1489396fe8a9eb87c0abf8aa5b2ef267fda # v2
```